cleanlab/datalab/internal/adapter/imagelab.py

```diff
@@ -11,7 +11,14 @@
 import pandas as pd
 from scipy.sparse import csr_matrix
 
-from cleanlab.datalab.internal.adapter.constants import DEFAULT_CLEANVISION_ISSUES
+from cleanlab.datalab.internal.adapter.constants import (
+    DEFAULT_CLEANVISION_ISSUES,
+    IMAGELAB_ISSUES_MAX_PREVALENCE,
+)
+from cleanlab.datalab.internal.adapter.constants import (
+    DEFAULT_CLEANVISION_ISSUES,
+    IMAGELAB_ISSUES_MAX_PREVALENCE,
+)
```

Comment:
These imports are duplicated


```suggestion
from cleanlab.datalab.internal.adapter.constants import (
    DEFAULT_CLEANVISION_ISSUES,
    IMAGELAB_ISSUES_MAX_PREVALENCE,
)
```

Reply:
apologize for this, still new to vscode merge